example_site/src/components/TemplateBrowser.js

```diff
@@ -86,20 +101,21 @@ function CollectionFilter({
   selectedCollection,
   setSelectedCollection,
 }) {
-  const buttonColor = selectedCollection === collection ? blueColor : blackColor;
+  const filterNameColor = selectedCollection === collection ? blueColor : blackColor;
+  const filterFontWeight = selectedCollection === collection ? '800' : '400';
 
   const onClick = (collection) => {
     setSelectedCollection(collection);
   };
 
   return (
-    <div className="category-name">
+    // <div className="category-name">
       <button className="filter-name" onClick={() => onClick(collection)}>
-        <div style={{'color': buttonColor}}>
+        <div style={{'color': filterNameColor, 'font-weight': filterFontWeight}}>
```

Comment:
Same comment about the format and converting hyphenated CSS properties to camelcase!